site/scripts/generate-candidates.mjs

```
@@ -0,0 +1,107 @@
+  return urls;
+}
+
+function toMarkdown(indexUrl, urls) {
+  const lines = [];
+  lines.push(`# Candidate poems (${urls.length})`);
+  lines.push('');
+  lines.push(`Source index: ${indexUrl}`);
+  lines.push('');
+  lines.push('One URL per line (ranked by appearance in the index HTML).');
+  lines.push('');
+  urls.forEach((u, i) => lines.push(`${String(i + 1).padStart(3, '0')}. ${u}`));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)